go/util/options/options.go

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2015, Kevin Walsh.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package options works in concert with flag, adding prettier printing of
+// options.
+package options
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	_ "github.com/golang/glog"
+)
+
+// Option is like flag.Flag, but supports prettier printing.
+type Option struct {
+	// Name for this flag, e.g. "pass"
+	Name string
+
+	// Default value, e.g. "BogusPassword"
+	Default interface{}
+
+	// Placeholder description of the argument e.g. "<password>"
+	Prototype string
+
+	// Help message, e.g. "Password for decryption"
+	Help string
+
+	// Relevance is a comma-separated list of words, used to group the flags
+	// into categories.
+	Relevance string
+}
+
+type Category struct {
```

Comment:
nit: please add a comment to the exported struct (and run go vet to warn you about others I might have missed)